TelemetryManager/Inc/c_library_v2/checksum.h

```diff
@@ -0,0 +1,95 @@
+#pragma once
```

Comment:
can we rename this folder to like something mavlink related, "c_library_v2" isnt very descriptive for our use case

Reply:
Right. That would make much more sense. Will do.

Reply:
Done ✅